src/core/meshcop/joiner.cpp

```diff
@@ -188,7 +192,7 @@ void Joiner::HandleDiscoverResult(otActiveScanResult *aResult)
         }
         else
         {
-            otLogDebgMeshCoP("Steering data not set");
+            otLogDebgMeshCoP("Steering data not allow the joining");
```

Comment:
nit: `Steering data not allow the joining` -> `Steering data does not include this device`.